src/scss/components/_form.scss

```diff
@@ -7,6 +7,34 @@
 **/
 
 .form {
+  position: relative;
```

Comment:
@iondrimba I do not understand why we have a position: relative? I did not find anything of properties like:
- left;
- right;
- bottom;
- top;
- relative: absolute;
- relative: fixed.